kt/entry-generation/godot-entry-generator/src/main/kotlin/godot/entrygenerator/filebuilder/ClassRegistrarFileBuilder.kt

```diff
@@ -56,11 +50,14 @@ class ClassRegistrarFileBuilder(
         .beginControlFlow("with(registry)") //START: with registry
         .let { funSpecBuilder ->
             if (!registeredClass.isAbstract) {
+                val superClasses = registeredClass.supertypes.takeWhile {
+                    //Used to implement script inheritance methods, so we remove base types and abstract parents.
+                    it.name != registeredClass.godotBaseClass && !it.isAbstract
+                }.map { "\"${it.name}\"" }.reduceOrNull{ acc, string -> "$acc,$string" }?:""
```

Comment:
I see two problems here:
- this will probably break when the user configures fqName registration as you only use the name
- when you use the fqName the imports are not automatically added. Use poet templates so the imports are correctly added (`%T`)